extensions/mssql/notebooks/TSG/cluster-status.ipynb

```diff
@@ -0,0 +1,122 @@
+{
```

Comment:
nitpick: let's use `tsg` for directory name if possible to minimize casing risks on different platforms.